src/components/structural/header/Header.js

```diff
@@ -139,21 +141,23 @@ class Header extends Component {
   */
   handleKeyDown(e) {
     //metaKey is cmd and windows key in some browsers
-    if ((e.ctrlKey || e.metaKey) && (e.key === "Enter" || e.key === "Return")) {
-      //ctrl/cmd + enter renders the scene
-      e.preventDefault();
-      this.clear();
-      this.handleRender();
-    } else if ((e.ctrlKey || e.metaKey) && e.shiftKey && (e.key === "s" || e.key === "S")) {
-      //ctrl/cmd + shift + s saves the scene with a new ID
-      e.preventDefault();
-      this.setState({ needsNewId: true });
-      this.handleSave();
-    } else if ((e.ctrlKey || e.metaKey) && (e.key === "s" || e.key === "S")) {
-      //ctrl/cmd + s saves the scene
-      e.preventDefault();
-      this.handleSave();
-      this.handleSaveClose();
+    if (this.props.layoutType !== layoutTypes.REFERENCE) {
```

Comment:
Is your autoformatter on? We keep having weird formatting changes and I don't know why. (It's not just you.)

Reply:
yeah i turned on auto-format on save, and it made a bunch of interesting formatting changes.

Reply:
I need to look into that and test it locally. Not sure what it keeps being an issue.

Reply:
btw this change i made starting on line 144 was made because i accidentally hit CTRL+S on the reference page and it crashed MYR so I disabled keybinds on the reference page

Reply:
Sorry. I didn't notice that. I thought it was only a formatting thing.

Reply:
that wasn't a formatting thing, but all the weird spacing changes to reference.js were. not sure why those happened lol